lib/i18n.rb

```diff
@@ -172,7 +174,7 @@ def translate(*args)
 
     # Wrapper for <tt>translate</tt> that adds <tt>:raise => true</tt>. With
     # this option, if no translation is found, it will raise <tt>I18n::MissingTranslationData</tt>
-    def translate!(key, options={})
+    def translate!(key, options = EMPTY_HASH)
       translate(key, options.merge(:raise => true))
```

Comment:
Calling `merge` on a frozen hash will not work:

```
irb(main):001:0> a = {}.freeze
=> {}
irb(main):002:0> a.merge!(raise: true)
RuntimeError: can't modify frozen Hash
	from (irb):2:in `merge!'
	from (irb):2
	from /Users/ryanbigg/.rubies/ruby-2.4.2/bin/irb:11:in `<main>'
```

Reply:
huh this means that it is not covered with tests, need to fix this

Reply:
Wait a second. Yes `.merge!` will raise an error, because you can not mutate object. But this code uses `.merge` without exclamation mark